js/backup_restore.js

```diff
@@ -636,7 +636,7 @@ function configuration_restore(callback) {
     function configuration_upload(configuration, callback) {
         function upload() {
             var activeProfile = null,
-                profilesN = 3;
```

Comment:
Ok. I am not in favour of deploying 'fixes' that break stuff for other users. Will submit a PR with proper version gating as soon as I get to it.